packages/mobile/src/services/libs.ts

```diff
@@ -1,12 +1,12 @@
 import EventEmitter from 'events'
 
 import type { Nullable } from '@audius/common'
-import type { AudiusLibs } from '@audius/sdk/dist/native-libs'
+import type { AudiusLibs } from '@audius/sdk'
 
 // TODO: declare this at the root and use actual audiusLibs type
 declare global {
   interface Window {
-    audiusLibs: any
+    audiusLibs: AudiusLibs
```

Comment:
🥲 thank you

Reply:
believe me i wasn't planning to do this work... it just happened 🌵